src/main/java/net/syscon/elite/api/resource/ScheduleResource.java

```diff
@@ -96,15 +105,22 @@
                                                                   @ApiParam(value = "Include suspended scheduled activity - defaults to false") @RequestParam(value = "includeSuspended", required = false) boolean includeSuspended);
 
     @PostMapping("/{agencyId}/activities")
-    @ApiOperation(value = "", nickname = "getActivities")
+    @ApiOperation(value = "", nickname = "getActivitiesForBookings")
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "", response = PrisonerSchedule.class, responseContainer = "List")})
-    List<PrisonerSchedule> getActivities(@ApiParam(value = "", required = true) @PathVariable("agencyId") String agencyId,
+    List<PrisonerSchedule> getActivitiesForBookings(@ApiParam(value = "", required = true) @PathVariable("agencyId") String agencyId,
                                          @ApiParam(value = "The required offender numbers (mandatory)", required = true) @RequestBody List<String> body,
                                          @ApiParam(value = "Date of whereabouts list, default today") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @RequestParam(value = "date", required = false) LocalDate date,
                                          @ApiParam(value = "AM, PM or ED", allowableValues = "AM,PM,ED") @RequestParam(value = "timeSlot", required = false) TimeSlot timeSlot,
                                          @ApiParam(value = "Whether to include 'excluded' activities in the results", defaultValue = "false") @RequestParam(value = "includeExcluded", required = false, defaultValue = "false") boolean includeExcluded);
 
+
+    @PostMapping("/activities")
+    @ApiOperation(value = "", nickname = "getActivities")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "", response = PrisonerSchedule.class, responseContainer = "List")})
```

Comment:
unnecessary 200 - will be generated automatically (esp. as aren't setting a custom message)

Reply:
The code seems to be a required parameter of ApiResponse. What did you have in mind? Happy to change this and the rest in this file.